firmware/applications/SignBuddy/sensors.h

```diff
@@ -7,4 +7,6 @@ void sensors_task_setup(void);
 void sensors_task_start(void);
 void sensors_sampling_timer_start(void);
 
+void sensors_sample_periodic(uint8_t);
```

Comment:
also add the parameter names in the header files, it's confusing without them

Reply:
Will do